deploy.py

```diff
@@ -22,9 +23,10 @@ def process(xcode, version):
       new_files = list(filter(lambda x : version in x, new_files))
 
   for i in new_files:
-    print 'Unzip file "{}.zip" to {}'.format(i, target)
+    print('Unzip file "', i, '.zip" to ', target, sep='')
     unzip_file(i, target)
-  print '\nUpdate successfully for {}'.format(xcode)
```

Comment:
Why change the code?  Just add the parens: `print('\nUpdate successfully for {}'.format(xcode))` because that is much easier to review and is type safe if xcode something other than `str`.

Reply:
are you suggesting not to support anything below v3 (since `print()` won't work in v2)? If that's the case then yes it makes sense to just keep it as a function

Reply:
The `__future__`directive __changes the way that Python 2 operates__:
```
% python2 -c "print('a', 'b')"
('a', 'b')

% python2 -c "from __future__ import print_function ; print('a', 'b')"
a b
```

Reply:
Thanks for the example, updated